contrib/gitian-descriptors/gitian-osx.yml

```diff
@@ -35,7 +35,7 @@ files:
 - "MacOSX10.15.sdk.tar.gz"
```

Comment:
You should be change `python` and `pythond-dev` to `python3` and `python3-dev` to fix compile error in gitian build.

```
Package python-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python2-dev python2 python-dev-is-python3

Package python is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  2to3 python2-minimal python2 dh-python python-is-python3

E: Package 'python' has no installation candidate
E: Package 'python-dev' has no installation candidate
```